packages/panels/docs/02-getting-started.md

```diff
@@ -111,11 +115,11 @@ class Treatment extends Model
 
 ## Introducing resources
 
-In Filament, resources are static classes that are used to build CRUD interfaces for your Eloquent models. They describe how administrators should be able to interact with data from your panel - using tables and forms.
+In Filament, resources are static classes used to build CRUD interfaces for your Eloquent models. They describe how administrators can interact with data from your panel using tables and forms.
 
-In our vet management panel, patients are at the core of all operations. Patients should be listed in a table, and clicking on a patient should open a new page where their details are editable. Patients should be able to be deleted from this page too. Above the patient list, there should be a button to allow the user to add a new patient to the system.
+Since patients (pets) are the core entity in this system, let's start by creating a patient resource that enables us to build screens for creating, viewing, updating, and deleting patients.
```

Comment:
```suggestion
Since patients (pets) are the core entity in this system, let's start by creating a patient resource that enables us to build pages for creating, viewing, updating, and deleting patients.
```